calling verify-copy.py
Verifying files for: arrow-alert-v1-1_psychopy
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000010/sub-3000010_arrow-alert-v1-1_s1_r1_e1.log[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000010/sub-3000010_arrow-alert-v1-1_s1_r1_e1.psydat[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000010/sub-3000010_arrow-alert-v1-1_s1_r1_e1.csv[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000014/sub-3000014_arrow-alert-v1-2_psychopy_s1_r1_e1_1.csv  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000014/sub-3000014_arrow-alert-v1-2_psychopy_s1_r1_e1_1.psydat  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: variable name arrow-alert-v1-2 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000026/sub-3000026_arrow-alert-v1-2_s1_r1_e1.psydat[0m
[31mError: variable name arrow-alert-v1-2 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000026/sub-3000026_arrow-alert-v1-2_s1_r1_e1.log[0m
[31mError: variable name arrow-alert-v1-2 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000026/sub-3000026_arrow-alert-v1-2_s1_r1_e1.csv[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000020/sub-3000020_arrow-alert-v1-1_s1_r1_e1.log[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000020/sub-3000020_arrow-alert-v1-1_s1_r1_e1.csv[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000020/sub-3000020_arrow-alert-v1-1_s1_r1_e1.psydat[0m
[31mError: number of psychopy data files in subject folder sub-3000008 5 greater than the expected number 3[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000008/extra-psychopy-present  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000008/sub-3000008_arrow-alert-v1-1_psychopy_s1_r1_e1_2.csv  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000008/sub-3000008_arrow-alert-v1-1_psychopy_s1_r1_e1_practice-only.csv  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
Verifying files for: arrow-alert-v1-2_psychopy
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000010/sub-3000010_arrow-alert-v1-1_s1_r1_e1.log[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000010/sub-3000010_arrow-alert-v1-1_s1_r1_e1.psydat[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000010/sub-3000010_arrow-alert-v1-1_s1_r1_e1.csv[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000014/sub-3000014_arrow-alert-v1-2_psychopy_s1_r1_e1_1.csv  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000014/sub-3000014_arrow-alert-v1-2_psychopy_s1_r1_e1_1.psydat  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: variable name arrow-alert-v1-2 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000026/sub-3000026_arrow-alert-v1-2_s1_r1_e1.psydat[0m
[31mError: variable name arrow-alert-v1-2 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000026/sub-3000026_arrow-alert-v1-2_s1_r1_e1.log[0m
[31mError: variable name arrow-alert-v1-2 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000026/sub-3000026_arrow-alert-v1-2_s1_r1_e1.csv[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000020/sub-3000020_arrow-alert-v1-1_s1_r1_e1.log[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000020/sub-3000020_arrow-alert-v1-1_s1_r1_e1.csv[0m
[31mError: variable name arrow-alert-v1-1 does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000020/sub-3000020_arrow-alert-v1-1_s1_r1_e1.psydat[0m
[31mError: number of psychopy data files in subject folder sub-3000008 5 greater than the expected number 3[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000008/extra-psychopy-present  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000008/sub-3000008_arrow-alert-v1-1_psychopy_s1_r1_e1_2.csv  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/psychopy/sub-3000008/sub-3000008_arrow-alert-v1-1_psychopy_s1_r1_e1_practice-only.csv  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
Verifying files for: all_audacity
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/audacity/sub-3000013/sub-3000013_all_audacity_s1_r1.zip.gpg  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/audacity/sub-3000023/sub-3000023_all_audacity_s1_r1.zip.gpg  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
Verifying files for: all_zoom
Verifying files for: all_eeg
[31mError: number of eeg data files in subject folder sub-3000013 6 greater than the expected number 3[0m
[31mError: variable name speech_eeg does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/eeg/sub-3000013/sub-3000013_speech_eeg_s1_r1_e1.vmrk[0m
[31mError: variable name speech_eeg does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/eeg/sub-3000013/sub-3000013_speech_eeg_s1_r1_e1.vhdr[0m
[31mError: variable name speech_eeg does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/eeg/sub-3000013/sub-3000013_speech_eeg_s1_r1_e1.eeg[0m
Verifying files for: all_digi
[31mError: variable name digi does not match any datadict variables, in file: /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/digi/sub-3000010/sub-3000010_digi_s1_r1_e1.zip.gpg[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/digi/sub-3000003/no-digi  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/digi/sub-3000006/no-digi  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/digi/sub-3000008/no-digi  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
[31mError: file  /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/digi/sub-3000011/no-digi  does not match naming convention <sub-#>_<variable/task-name>_<session>.<ext>[0m
Accessing /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap
	 Newest Redcap found: Thrivebbschilds1r1_DATA_2023-09-14_1237.csv
	 redcap/Thrivebbschilds1r1_DATA_2023-09-14_1237.csv already exists in checked, skipping copy 

	 Newest Redcap found: Thrivebbsparents1r1_DATA_2023-09-14_1238.csv
	 redcap/Thrivebbsparents1r1_DATA_2023-09-14_1238.csv already exists in checked, skipping copy 

	 Newest Redcap found: ThrivebbsRAs1r1_DATA_2023-09-14_1238.csv
	 redcap/ThrivebbsRAs1r1_DATA_2023-09-14_1238.csv already exists in checked, skipping copy 

	 Newest Redcap found: Thriveconsent_DATA_2023-09-14_1238.csv
	 redcap/Thriveconsent_DATA_2023-09-14_1238.csv already exists in checked, skipping copy 

	 Newest Redcap found: Thriveiqschilds1r1_DATA_2023-09-14_1239.csv
	 redcap/Thriveiqschilds1r1_DATA_2023-09-14_1239.csv already exists in checked, skipping copy 

	 Newest Redcap found: Thriveiqsclinicians1_DATA_2023-09-14_1237.csv
	 redcap/Thriveiqsclinicians1_DATA_2023-09-14_1237.csv already exists in checked, skipping copy 

	 Newest Redcap found: Thriveiqsparents1r1_DATA_2023-09-14_1239.csv
	 redcap/Thriveiqsparents1r1_DATA_2023-09-14_1239.csv already exists in checked, skipping copy 

updating tracker, ses: s1_r1/, redcaps: Thrivebbschilds1r1_DATA_2023-09-14_1237.csv Thrivebbsparents1r1_DATA_2023-09-14_1238.csv ThrivebbsRAs1r1_DATA_2023-09-14_1238.csv Thriveconsent_DATA_2023-09-14_1238.csv Thriveiqschilds1r1_DATA_2023-09-14_1239.csv Thriveiqsclinicians1_DATA_2023-09-14_1237.csv Thriveiqsparents1r1_DATA_2023-09-14_1239.csv
Traceback (most recent call last):
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexes/base.py", line 3080, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 70, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 96, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 126, in pandas._libs.index.IndexEngine._get_loc_duplicates
  File "pandas/_libs/index.pyx", line 133, in pandas._libs.index.IndexEngine._maybe_get_bool_indexer
  File "pandas/_libs/index.pyx", line 150, in pandas._libs.index.IndexEngine._unpack_bool_indexer
KeyError: 'id'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/data/NDClab/datasets/thrive-dataset/data-monitoring/update-tracker.py", line 127, in <module>
    ids = get_IDs(df_dd)
  File "/home/data/NDClab/datasets/thrive-dataset/data-monitoring/update-tracker.py", line 74, in get_IDs
    id_desc = df_dd.set_index("variable").loc["id", "description"].split(" ")
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexing.py", line 889, in __getitem__
    return self._getitem_tuple(key)
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexing.py", line 1060, in _getitem_tuple
    return self._getitem_lowerdim(tup)
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexing.py", line 807, in _getitem_lowerdim
    section = self._getitem_axis(key, axis=i)
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexing.py", line 1124, in _getitem_axis
    return self._get_label(key, axis=axis)
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexing.py", line 1073, in _get_label
    return self.obj.xs(label, axis=axis)
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/generic.py", line 3739, in xs
    loc = index.get_loc(key)
  File "/home/applications/spack/applications/gcc-8.2.0/miniconda3-4.5.11-oqs2mbgv3mmo3dll2f2rbxt4plfgyqzv/lib/python3.7/site-packages/pandas/core/indexes/base.py", line 3082, in get_loc
    raise KeyError(key) from err
KeyError: 'id'
args: /home/data/NDClab/datasets/thrive-dataset/data-monitoring/update-tracker.py /home/data/NDClab/datasets/thrive-dataset/sourcedata/checked /home/data/NDClab/datasets/thrive-dataset /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/Thrivebbschilds1r1_DATA_2023-09-14_1237.csv,/home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/Thrivebbsparents1r1_DATA_2023-09-14_1238.csv,/home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/ThrivebbsRAs1r1_DATA_2023-09-14_1238.csv,/home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/Thriveconsent_DATA_2023-09-14_1238.csv,/home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/Thriveiqschilds1r1_DATA_2023-09-14_1239.csv,/home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/Thriveiqsclinicians1_DATA_2023-09-14_1237.csv,/home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s1_r1/redcap/Thriveiqsparents1r1_DATA_2023-09-14_1239.csv s1_r1 true

No redcap folder found in /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s2_r1/ directory
No redcap folder found in /home/data/NDClab/datasets/thrive-dataset/sourcedata/raw/s3_r1/ directory
